pikaur/config.py

```diff
@@ -151,7 +151,7 @@ def init_value(cls) -> Path:
             or os.environ.get(
                 "XDG_CACHE_HOME",
             )
-            or Home()() / ".cache/",
+            or Path(PikaurConfig().misc.CachePath.get_str()),
```

Comment:
the order is wrong - if user have XDG_CACHE_HOME set - the config value would be ignored.

Reply:
or otherwise, the documentation should be updated to mention that config option would be overriden by this specific env var